README.md

```diff
@@ -28,5 +28,13 @@ downloads it from Github if it changed since the last run. If you want to avoid
 the HTTP round-trip, use `--local`. To force a database update without checking
 for a project, use `--update-cache`.
 
+Building Docker image
+
+    $ docker build -t local-php-security-checker
```

Comment:
```suggestion
    $ docker build -t local-php-security-checker .
```

Reply:
To build from project's root folder, docker build command requires the path to the dockerfile